### DISCUSSION TITLE
Release PR about LINQ2

### DISCUSSION THREAD

**Discussion by SpartanNico94** [elsa-workflows/elsa-core#4048] [Q&A] [answered]
Is there already a planned date when this [PR](https://github.com/elsa-workflows/elsa-core/pull/4001) will be available as NuGet Package?

**Comment by mohdali** [comment, marked as answer]
It is released today in 2.12.0 https://github.com/elsa-workflows/elsa-core/releases/tag/2.12.0

**Comment by SpartanNico94** [comment]
Oh Nice!! Thanks for the support
